packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -557,6 +556,22 @@ export interface CallWithChatAdapterManagement {
    * Return to origin call of breakout room
    */
   returnFromBreakoutRoom(): Promise<void>;
+  /* @conditional-compile-remove(media-access) */
+  forbidAudio: (userIds: string[]) => Promise<void>;
```

Comment:
Like the CallAdapter lets look to see if we can be more type safe here with the SDK identifier type

Reply:
![image](https://github.com/user-attachments/assets/6b185c43-3e92-4d87-8c7a-55711ebba17c)
Will follow the existing pattern for now but agree with you to use strong typed user identifier